apps/server/src/modules/files/uc/delete-files.uc.ts

```diff
@@ -1,11 +1,14 @@
 /* eslint-disable no-await-in-loop */
 import { DeleteObjectCommand, S3Client } from '@aws-sdk/client-s3';
 import { Injectable } from '@nestjs/common';
-import { File, StorageProvider } from '@shared/domain';
-import { FilesRepo } from '@shared/repo';
+
+import { StorageProviderEntity } from '@shared/domain';
 import { StorageProviderRepo } from '@shared/repo/storageprovider/storageprovider.repo';
 import { LegacyLogger } from '@src/core/logger/legacy-logger.service';
 
+import { FilesRepo } from '../repo';
+import { FileEntity } from '../entity';
```

Comment:
Please remove new rows and let them sort automatically

Reply:
Let's continue the discussion here https://github.com/hpi-schul-cloud/schulcloud-server/pull/4303#discussion_r1305536299 .